aqueduct_client/aqueduct_client.py

```diff
@@ -0,0 +1,243 @@
+import datetime
+
+import requests
+import pandas as pd
+
+from .utils import load_api_key
+
+
+class AqueductClient(object):
+    def __init__(self, api_key=None, base_url=None):
+        """
+        Creates an AqueductClient.
+
+        Parameters
+        ----------
+        api_key : str, optional
+            The Quantopian API key to use.  If not given, we attempt
+            to load the key from a credentials file or from an
+            environment variable.
+
+        base_url : str, optional
+            The base URL for the Aqueduct API.  Defaults to the
+            FactSet Aqueduct endpoint.
+        """
+        if base_url:
+            self._base_url = base_url
+        else:
+            self._base_url = \
+                "https://factset.quantopian.com/api/experimental/pipelines"
```

Comment:
We probably should be versioning this API url.

Reply:
entirely agree, but out of scope of this PR.  we should start versioning asap